setup.py

```diff
@@ -0,0 +1,58 @@
+# This file is part of ts_ess_common.
+#
+# Developed for the Vera Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import setuptools
+from typing import List
+
+install_requires: List[str] = []
+tests_require: List[str] = [
+    "pytest",
+    "pytest-cov",
+    "pytest-flake8",
+    "pytest-black",
+    "pytest-mypy",
+]
```

Comment:
I think we are no longer using the `tests_require` argument of `setuptools.setup`.

In fact you define the value here but then don't use it. I'm surprised flake8 doesn't catch this (but it certainly didn't for me, and presumably doesn't for you either).

Reply:
I removed that. Note that this also was in

https://confluence.lsstcorp.org/display/LTS/Enabling+Mypy+in+Pytest

so I removed it there as well.